templates/quickstart-confluence-master.template.yaml

```diff
@@ -1414,6 +1448,10 @@ Resources:
                     - "ATL_RELEASE_S3_BUCKET=atlassian-software"
                     - "ATL_RELEASE_S3_PATH=releases/confluence"
                     - "ATL_SSL_SELF_CERT_ENABLED=false"
+                    - "ATL_SYNCHRONY_DATABASE_IDLE_CONNECTION_TEST_PERIOD=100"
```

Comment:
We'll need to confirm these settings for C3P0. Leaving this comment here to verify the settings

Reply:
Will also require an ami-rebake at some point after we confirm there is no further work to be done in the scripts.